### PR TITLE
Hebrew support

### PR DESCRIPTION
**Pull request by yaccri**

### CODE REVIEW

File: src/localize/languages/he.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        "no_entries": "אין פריטים להצגה",
+        "backend_error": "אין אפשרות להתחבר לרכיב התזמונים. נדרש להתקין את הרכיב באינטגרציה לפני השימוש בכרטיס.",
+        "excluded_items": "{number} לא נכלל {if number is 1} פריט {else} פריטים",
+        "hide_excluded": "הסתר פריטים לא כלולים",
+        "additional_tasks": "{number} נוסף {if number is 1} משימה {else} משימות"
+      },
+      "entity_picker": {
+        "no_groups_defined": "לא הוגדרו קבוצות",
+        "no_group_selected": "בחר קבוצה תחילה",
+        "no_entities_for_group": אין יישויות בקבוצה זו",
```
Comment (nielsfaber) [inline src/localize/languages/he.json:110]: Missing a bracket here

File: src/localize/languages/he.json [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        "any": "אחד מהתנאים",
+        "no_conditions_defined": "לא הוגדרו תנאים"
+      },
+      "options": {
+        "repeat_type": "התנהגות לאחר הפעלה"
+      }
+    }
+  }
+}
+© 2021 GitHub, Inc.
```
Comment (nielsfaber) [inline src/localize/languages/he.json:132]: All below closing bracket should be removed.

**Review by nielsfaber — CHANGES_REQUESTED** [review]

**Comment by nielsfaber** [comment]
@yaccri  can you update this pull request with the changes I've commented?

File: src/localize/languages/he.json (hunk 1 above)
Comment (yaccri) [reply src/localize/languages/he.json:110]: "no_entities_for_group": "אין יישויות בקבוצה זו",

Comment (yaccri) [reply src/localize/languages/he.json:110]:
```suggestion
        "no_entities_for_group": "אין יישויות בקבוצה זו",

```

**Comment by yaccri** [comment]
I'm not sure if it's corrected as I intended.
Line 110 had a missing quote and should be:         "no_entities_for_group": אין יישויות בקבוצה זו",

**Comment by nielsfaber** [comment]
Good enough for me 👍 
Thanks again!

**Pull request merged**